### ISSUE TITLE
How can I tell which npm package an icon is in?

### ISSUE THREAD

**Issue by Ethan-Arrowood** [FortAwesome/react-fontawesome#45]
Is there any place on the main website to find out which npm package an icon is located in?

**Comment by NateRadebaugh** [comment]
@Ethan-Arrowood You can apply a variety of filters using the website: https://fontawesome.com/icons?d=gallery

You can see here whether it is Solid, Regular, Light, or Brand.

At https://fontawesome.com/how-to-use/js-component-packages you can see the different npm packages:

**Comment by swrobel** [comment]
It'd be great if that was visible on an individual icon's page. For example, there's no indication that this belongs in brands rather than solid: https://fontawesome.com/icons/app-store-ios

**Comment by NateRadebaugh** [comment]
Good point. The only indicator is the class on the example code: `<i class="fab fa-app-store-ios"></i>` Where `fab` indicates it's from the brands set.

**Comment by talbs** [comment]
Thanks for bringing this up, folks. I agree, that seems like a huge pain. We'll see what we can do about getting some better metadata on the icon detail page (if not within our listings views).

Hang tight and sorry for the headaches in the meantime! 😄
